docs/guides/admin/docs/releasenotes.md

```diff
@@ -1,5 +1,17 @@
 # Opencast 12: Release Notes
 
+Opencast 12.2
+-------------
+
+The second maintenance release of Opencast 12.
+Notable changes:
+
+- New Opencast Studio version including dark mode feature
+- Fix OAI-PMH
+- Update CAS documentation
+
```

Comment:
```suggestion
- Bugfix update the MariaDB database driver (again)
- Use an event title fallback from Dublin Core catalog while publishing to
  Engage
- Update the migration script installation instructions, which will otherwise
  fail with newer versions of the MariaDB Python connector

```